test/functional/ws/transport.transactions.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var node = require('../../node');
+var shared = require('../http/post/shared');
+var ws = require('../../common/wsCommunication');
+
+function postTransaction (transaction, done) {
+	ws.call('postTransactions', {
+		transaction: transaction
+	}, done, true);
+}
+
+describe('Basic Transactions via websockets', function () {
```

Comment:
`Posting transaction (type 0)`

For other `ws/transport` tests, websockets is implied.